Resources/Prototypes/Corvax/Roles/play_time_trackers.yml

```
@@ -0,0 +1,11 @@
+- type: playTimeTracker 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)